Dockerfile

```diff
@@ -94,9 +103,20 @@ RUN chmod +x /usr/local/bin/configure.sh && \
 COPY deploy-screens/index.html /var/www/html/index.html
 
 ## Copy the application code
-COPY . /var/www/dms/
-
-COPY --from=0 /video-processing-cli/ "/var/www/dms/bin/"
+COPY \
```

Comment:
This is not enough as `composer install` is not performed. These are the required steps https://github.com/k-box/k-box/blob/master/.gitlab-ci.yml#L80-L90.
Maybe can be executed in a pre-build step

Reply:
Seperated into new build step